commands/roles.go

```diff
@@ -0,0 +1,75 @@
+package commands
+
+import (
+	"strings"
+	"errors"
```

Comment:
```suggestion
```
My bad, remove this. I was going to use it to throw an error if there was no such role in the server (i.e. if roleID was not set), but that should be the responsibility of the programmer to ensure it exists.